chart/Chart.yaml

```diff
@@ -1,24 +1,9 @@
 apiVersion: v2
 name: vcluster-rancher-op
-description: A Helm chart for Kubernetes
+description: Helm chart for vCluster Rancher Operator
 
-# A chart can be either an 'application' or a 'library' chart.
-#
-# Application charts are a collection of templates that can be packaged into versioned archives
-# to be deployed.
-#
-# Library charts provide useful utilities or functions for the chart developer. They're included as
-# a dependency of application charts to inject those utilities and functions into the rendering
-# pipeline. Library charts do not define any templates and therefore cannot be deployed.
 type: application
 
-# This is the chart version. This version number should be incremented each time you make changes
-# to the chart and its templates, including the app version.
-# Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.0
+version: 0.0.1-rc1
 
-# This is the version number of the application being deployed. This version number should be
-# incremented each time you make changes to the application. Versions are not expected to
-# follow Semantic Versioning. They should reflect the version the application is using.
-# It is recommended to use it with quotes.
-appVersion: "1.16.0"
+appVersion: "0.0.1-rc1"
```

Comment:
Should probably automate this part long term, to look at tag

Reply:
Can do it in the future. I feel like appVersion is not super useful when there is a 1 to 1 mapping between chart version and image version.